src/main/webapp/app/shared/subject/subject-pair-dialog.component.html

```diff
@@ -10,39 +10,39 @@
     </div>
     <div class="modal-body">
         <div class="form-group">
-            <div>
-                <label for="login" class="form-control-label"
-                       [translate]="'managementPortalApp.subject.login'">
-                </label>
-                <input id="login" type="text" class="form-control" name="login" #loginInput="ngModel"
-                       [(ngModel)]="subject.login" [disabled]="subject.id" >
-            </div>
+            <div *ngIf="(siteSettings$ | async).hiddenSubjectFields as hiddenFields">
```

Comment:
Instead of adding an additional div layer, you can add this `ngIf` to the `<div class="form-group">` instead. Or only add this additional div layer to the properties that are actually modified by the siteSettings.